.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.languageServer": "Default"
```

Comment:
I think you should add this file in the gitignore file so it won't pollute your diff, as it's specific of you
I see that this is removed in following commit.
Exercise for you, try to fix this in this commit 'Chapter 1-2 - First commits") instead of creating another commit to do so :)